src/client/actions/types.js

```diff
@@ -5,3 +5,5 @@ export const UPDATE_ROOM_LIST = 'UPDATE_ROOM_LIST';
 export const UPDATE_ROOM_USERS = 'UPDATE_ROOM_USERS';
 
 export const GET_ROOM_DETAILS = 'GET_ROOM_DETAILS';
+
+export const START_GAME = 'START_GAME';
```

Comment:
Actions/types should live close to their reducers where possible - might be time to split this out? I assume the spaces are because they relate to different reducers